README.md

```diff
@@ -0,0 +1,112 @@
+# cpy-rs: Python and C++ Multibinding Export Macro
+
+![cpy-rs Logo](https://github.com/patrickelectric/cpy-rs/blob/main/cpy-rs-logo.png)
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Features](#features)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+  - [Usage](#usage)
+- [Examples](#examples)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Introduction
+
+cpy-rs is a powerful macro that simplifies the deployment of libraries to both Python and C++. 
+This project allows you to create a multibinding export core, making easier for developers to create and distribute libraries that are already made in Rust, but now making them accessible to a broader range of users.
+
+This README provides essential information on how to get started with cpy-rs, its key features, installation guidelines, and usage examples.
+
+## Features
+
+- Easy deployment of libraries to Python and C++ environments.
+- Simplified multibinding export core for seamless integration.
+- Empowers developers to distribute Python and C++ libraries with minimal effort.
+- Supports a wide range of use cases, making it versatile for different projects.
+
+## Getting Started
+
+### Prerequisites
+
+Before using cpy-rs, ensure you have the following prerequisites installed on your system:
+
+- Rust 
+- Python
+- C++ compiler
+
+### Installation
+
+To use cpy-rs in your project, run `cargo add cpy-binder`, or include it as a dependency in your Rust project's `Cargo.toml` file:
+
+```toml
+[dependencies]
+cpy-binder = "0.3.0"
+
+### Usage
+
+1. Import the `cpy-binder` macro into your Rust project:
+
+```rust
+#[cfg(feature = "python")]
+use pyo3::{prelude::*, wrap_pyfunction};
+
+use cpy_binder::export_cpy;
+```
+
+2. Initiate your Rust module collections like following.
+
+```rust
+// Rust function using cpy macro
+export_cpy!(
+    mod my_module {
+        fn return_true() -> bool {
+            let x = true
+        }
+    }
+```
+
+3. Build your Rust project with the `cpy` feature enabled:
+
+```bash
+cargo build
```

Comment:
It's missing toolchain configuration for nightly build, project do not build with current stable.